website/markdown/docs/commands/dependencies.mdx

```diff
@@ -0,0 +1,211 @@
+---
+name: Manage third-party dependencies
+excerpt: "Learn how to use Tuist's dependencies command to manage third-party dependencies."
+---
+
+import PropertiesTable from '../components/properties-table'
+import SettingsDictionaryTable from '../components/settingsdictionary-table'
+import Message from '../components/message'
+import EnumTable from '../components/enum'
+import {
+  List,
+  ListItem,
+  ListIcon,
+  ListContent,
+  ListHeader,
+  ListDescription,
+  ListList,
+} from '../components/list'
+
+# Dependencies (Chimera)
+
+## How to get started
+
+1. List requirements for third-party dependencies in `Dependencies.swift` manifest.
+2. Install dependencies using `tuist dependencies [subcommand]` command.
+3. Link dependencies with project's targets.
+4. Regenerate project.
+
+## `Dependencies.swift`
+
+Requirements for third-party dependencies are defined in `Dependencies.swift` manifest. `Dependencies.swift` need to import the framework `ProjectDescription` which contains all the classes and enums that are available for you to describe your projects.
+
+You can edit `Dependencies.swift` using `tuist edit` command.
+
+Tuist expects `Dependencies.swift` under `Tuist/` directory:
```

Comment:
Hmm, I have always used *in* directory - but it seems both are acceptable, though in seems to be more common 😛